src/utils/helpers.js

```diff
@@ -0,0 +1,25 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = "SAVE",
+  loadingText = "SAVING..."
+) {
+  if (isLoading) {
+    btn.textContent = loadingText;
+  } else {
+    btn.textContent = defaultText;
+  }
+}
+
+export function setSubmitText(
```

Comment:
Please, delete it


* `setSubmitText` is not needed because it duplicates the same `setButtonText` function. To make it work for the deleting modal, you need to pass the 3rd and 4th arguments